config/config_test.go

```diff
@@ -352,7 +352,7 @@ systemd:
 							Name:     "test1.service",
 							Enable:   true,
 							Contents: "test1 contents",
-							DropIns: []types.SystemdUnitDropIn{
```

Comment:
This is an artifact of the json schema change. It wasn't missed, 2.0.0 didn't use json schema. Since this PR is moving to 2.1, this PR is moving to the autogenerated json schema structs in Ignition, which results in naming changes like this.